packages/common/src/index.ts

```diff
@@ -359,6 +360,58 @@ export default class Common {
     }
   }
 
+  /**
+   * Internal helper function to calculate a fork hash
+   * @param hardfork Hardfork name
+   * @returns Fork hash as hex string
+   */
+  _calcForkHash(hardfork: string) {
+    const genesis = Buffer.from(this.genesis().hash.substr(2), 'hex')
+
+    function pad(str: string, max: number): string {
+      return str.length < max ? pad('0' + str, max) : str
+    }
+
+    let hfBuffer = Buffer.alloc(0)
+    let prevBlock = 0
+    for (const hf of this.hardforks()) {
+      const block = hf.block
+
+      // Skip for chainstart (0), not applied HFs (null) and
+      // when already applied on same block number HFs
+      if (block !== 0 && block !== null && block !== prevBlock) {
+        const hfBlockBuffer = Buffer.from(pad(block.toString(16), 16), 'hex')
```

Comment:
Replace by native [`String.prototype.padStart()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)
```suggestion
        const hfBlockBuffer = Buffer.from(block.toString(16).padStart(16, '0'), 'hex')
```